http.go

```diff
@@ -281,6 +287,31 @@ func (w *requestBodyWriter) Write(p []byte) (int, error) {
 	return len(p), nil
 }
 
+func (resp *Response) parseNetConn(conn net.Conn) {
+	if conn.RemoteAddr() != nil {
+		if resp.raddr == nil {
+			resp.raddr = make([]byte, 0)
```

Comment:
You can safe one allocation by already reserving the amount of bytes you're going to need below.